accelerator/config.c

```diff
@@ -126,6 +126,15 @@ status_t cli_core_set(ta_core_t* const core, int key, char* const value) {
       }
       iota_service->http.port = ta_conf->iota_port_list[0];
       break;
+    case START_FILE:
+      if (value != NULL) {
+        if ((file = fopen(value, "w")) == NULL) {
+          ta_log_error("%s\n", ta_error_to_string(SC_CONF_FOPEN_ERROR));
+          return SC_CONF_FOPEN_ERROR;
+        }
+        fclose(file);
```

Comment:
I din't get the reason why you close the file immediately after its open.

Reply:
I just want to create a file to notify the regression runner that TA has been successfully initialized, but as you said IPC is a much better and elegant solution